src/statistics_tracking/ReproductionStatistic.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Components;
+using DefaultEcs;
+using Newtonsoft.Json;
+
+public class ReproductionStatistic : IStatistic
+{
+    public StatsTrackerEvent LinkedEvent { get; set; } = StatsTrackerEvent.PlayerReproduced;
+
+    [JsonProperty]
+    public int TimesReproduced { get; set; }
+
+    [JsonProperty]
+    public Dictionary<string, int> ReproducedInBiomes { get; set; } = new();
```

Comment:
We don't use the approach of storing names of things, the real instances are used instead. I suspect that there's some saving trouble? If so could you prep this to use the instances as keys and I can dive into the save format problems?